GetJobber/atlantis
# fix(components): make autocomplete work in input groups

## Motivations

This allows Autocomplete components to be used inside of InputGroup components and work properly with the layout.



## Changes

### Added

- Autocomplete can now be used inside of InputGroups.

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

1. View the test page: http://localhost:3333/components/input-group-test or https://app.netlify.com/sites/jobber-atlantis/deploys/5f32d73b9699e40008338fbe/components/input-group-test
2. Observe that the Autocomplete displays correctly in the two examples.
3. Feel free to test out any edge cases in the playground code.



---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

![Random photo of Atlantis](https://loremflickr.com/672/400/atlantis)
